apps/main/src/loan/components/PageLoanManage/Page.tsx

```diff
@@ -67,13 +63,13 @@ const Page = () => {
   const [loaded, setLoaded] = useState(false)
 
   const isValidRouterParams = !!rChainId && !!rCollateralId && !!rFormType
-  const isReady = !!curve?.signerAddress && !!llamma
+  const isReady = !!curve?.signerAddress && !!market
 
-  const marketDetails = useMarketDetails({ chainId: rChainId, llamma, llammaId })
+  const marketDetails = useMarketDetails({ chainId: rChainId, llamma: market, llammaId: marketId })
   const positionDetails = useLoanPositionDetails({
     chainId: rChainId,
-    llamma,
-    llammaId,
+    llamma: market,
+    llammaId: marketId,
```

Comment:
we should decide how to name these parameters, you are refactoring to marketId which is fine, but we also use poolId, llammaId and whatever else